RoomLocator/RoomLocator.Domain/Models/Feedback.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace RoomLocator.Domain.Models
+{
+    public class Feedback
```

Comment:
1. Der mangler id
2. Hvordan kan du se hvem der har upvoted/downvoted?
3. Hvordan kan du se hvornår der blev upvoted/downvoted?
4. Hvad sker der hvis Upvote og Downvote begge er true? Eller begge er false?

Upvote/downvote bør være én variabel, for at undgå inkonsistent data

Reply:
Jeg ville nok gemme den bruger den er oprettet som, samt hvornår brugeren har oprettet den.